tests/test_odb.py

```diff
@@ -17,18 +17,28 @@ def test_odb(memfs):
     assert hash(odb) == hash(odb)
 
 
+def test_odb_init(memfs):
+    odb = ObjectDB(memfs, "/odb")
```

Comment:
What if we add something like `odb = ObjectDB(memfs, "/odb", init=True)` for easier testing.

Reply:
See https://github.com/iterative/dvc-objects/pull/147 , I don't think we need this anymore.